Snowflake.Data/Core/SFSessionProperty.cs

```diff
@@ -65,20 +65,19 @@ internal static SFSessionProperties parseConnectionString(String connectionStrin
                             SFSessionProperty p = (SFSessionProperty)Enum.Parse(

```

Comment:
A couple lines above, we split using the String.Split overload that takes a string array but pass in just a character. We should use the overload that takes a char[]. It will be slightly faster.